src/Nancy/Json/Simple/NancySerializationStrategy.cs

```diff
@@ -0,0 +1,217 @@
+﻿namespace Nancy.Json.Simple
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using Nancy.Extensions;
+
+    internal class NancySerializationStrategy : PocoJsonSerializerStrategy

```

Comment:
Users calling Deserialize, is not our problem. And if they do, it may be, because they want consistency rather than introducing json.net to serialize/deserialize since the Nancy impl was marked internal making it useless to the outside world. 

my 2c